templates/statefulset.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ .Release.name}}
+spec:
+  selector:
+    matchLabels:
+      app: {{ .Release.Name}}
+  serviceName: {{ .Values.service}}-headless
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: {{ .Release.Name}}
+    spec:
+      containers:
+      - name: pyrsia-node
+        image: {{.Values.image.pyrsia}}
+        ports:
+        - containerPort: {{.Values.service.pyrsia.port}}
+          name: pyrsia-node
+        - containerPort: {{.Values.service.libp2p.port}}
+          name: libp2p-node
+        volumeMounts:
+        - name: pyrsia
+          mountPath: {{.Values.mountpath}}
+  volumeClaimTemplates:

```

Comment:
Volume claim is already defined.  Let me provide the details in a separate comment